Aztec/Classes/Public/Extensions/NSLayoutManager+Attachments.swift

```diff
@@ -0,0 +1,40 @@
+import UIKit
+
+extension NSLayoutManager
+{
+    /// Determine the character ranges for an attachment
+    public func ranges(forAttachment attachment: NSTextAttachment) -> [NSRange]
```

Comment:
This method looks like a better fit for `NSAttributedString+Attachments.string`, since it only interacts with the attributed string.

`invalidateLayoutForAttachment()` could access this method through the `textStorage`.